src/explore-education-statistics-admin/src/services/configurationService.ts

```diff
@@ -1,14 +1,17 @@
 import { ApplicationInsights } from '@microsoft/applicationinsights-web';
 import client from '@admin/services/util/service';
 
-const service = {
+export const configurationService = {
   getInsightsKey(): Promise<string> {
     return client.get<string>('/configuration/application-insights');
   },
+  getPublicBaseUrl(): Promise<string> {
+    return client.get<string>('/configuration/public-app-url');
```

Comment:
This should be reimplemented as a proper REST endpoint and not a series of endpoints that provide individual variables (as mentioned on Friday).